catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/BroadcastBrickMessageUpdateTest.java

```diff
@@ -1,207 +0,0 @@
-	}
-
-	@Before
-	public void setUp() throws Exception {
-		createTestProjectWithBricks(BroadcastBrickMessageUpdateTest.class.getSimpleName());
-		baseActivityTestRule.launchActivity();
-	}
-
-	@Test
-	public void testAllBroadcastBrickSpinnersContainTheNewAddedMessage() {
-		createNewBroadcastMessageOnBrick(message, firstBroadcastBrick,
-				baseActivityTestRule.getActivity());
-
-		List<String> spinnerValues = Arrays.asList(
-				UiTestUtils.getResourcesString(R.string.new_option),
-				UiTestUtils.getResourcesString(R.string.edit_option),
-				defaultMessage,
-				message);
-
-		checkAllBrickSpinnerValues(spinnerValues);
-	}
-
-	@Test
-	public void testAllBroadcastBrickSpinnersContainTheEditedMessage() {
-		editBroadcastMessageOnBrick(defaultMessage, editedMessage, firstBroadcastBrick,
-				baseActivityTestRule.getActivity());
-
-		List<String> spinnerValues = Arrays.asList(
-				UiTestUtils.getResourcesString(R.string.new_option),
-				UiTestUtils.getResourcesString(R.string.edit_option),
-				editedMessage);
-
-		checkAllBrickSpinnerValues(spinnerValues);
-	}
-
-	@Test
-	public void testAllBroadcastBrickSpinnersShowTheEditedMessage() {
-		editBroadcastMessageOnBrick(defaultMessage, editedMessage, firstBroadcastBrick,
-				baseActivityTestRule.getActivity());
-
-		checkShowsCorrectSpinnerMessage(editedMessage);
-	}
-
-	@Test
-	public void testEditingOccursOnlyInCurrentScene() {
-		editBroadcastMessageOnBrick(defaultMessage, editedMessage, firstBroadcastBrick,
-				baseActivityTestRule.getActivity());
-
-		switchScene();
-
-		checkShowsCorrectSpinnerMessage(defaultMessage);
-	}
-
-	private void checkAllBrickSpinnerValues(List<String> spinnerValues) {
-		onBrickAtPosition(1)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkNameableValuesAvailable(spinnerValues);
-		onBrickAtPosition(2)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkNameableValuesAvailable(spinnerValues);
-		onBrickAtPosition(3)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkNameableValuesAvailable(spinnerValues);
-		onBrickAtPosition(4)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkNameableValuesAvailable(spinnerValues);
-		onBrickAtPosition(5)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkNameableValuesAvailable(spinnerValues);
-	}
-
-	private void checkShowsCorrectSpinnerMessage(String message) {
-		onBrickAtPosition(1)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkShowsText(message);
-		onBrickAtPosition(2)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkShowsText(message);
-		onBrickAtPosition(3)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkShowsText(message);
-		onBrickAtPosition(4)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkShowsText(message);
-		onBrickAtPosition(5)
-				.onSpinner(R.id.brick_broadcast_spinner)
-				.checkShowsText(message);
-	}
-
-	private void switchScene() {
-		baseActivityTestRule.finishActivity();
```

Comment:
I think the last activity is never finished this way, or is it?

Would that possibly cause any problems?